### PR TITLE
Teshari stuff

### PR DESCRIPTION
**Pull request by NeverKnowWhyThis**
-Adds a head to belly colouration for Teshari body marking
-Adds a smaller feathered alternative body marking
-Adds a Teshari mohawk hairstyle
-Adds a second option for Teshari tail (colorable)
~~-Adds a snowflake tail for Schae Shooki~~
-Adds 3 options to have ears on a Teshari

This was made reverse engineering a lot of old edits and asking the coders, mistakes are bound to have been made.

### CODE REVIEW

**Comment by Arokha** [comment]
Also do you have any sort of SCA or anything for `-Adds a snowflake tail for Schae Shooki`? 👀

File: code/modules/mob/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -1025,6 +1025,11 @@
 		icon_state = "teshari_mushroom"
 		species_allowed = list(SPECIES_TESHARI)
 
+	teshari_fluffymohawk
+		name = "Teshari Fluffy Mohawk"
+		icon_state = "teshari_fluffymohawk"
```
Comment (Arokha) [inline code/modules/mob/new_player/sprite_accessories.dm]: Also these need //VOREStation edit comments

Comment (Cameron653) [reply code/modules/mob/new_player/sprite_accessories.dm]: I'd personally suggest moving them to sprite_accessories.dm altogether.

**Comment by NeverKnowWhyThis** [comment]
Sprites need serious editing.

**Comment by KasparoVy** [comment]
I agree. For being in `..._add`, the mohawk sprite is very bright. The darkest pixel should be `rgb(0,0,0)`

**Comment by Ketrai** [comment]
this makes me wanna work on getting the last suits still missing vulpakanin sprites sprited

**Comment by NeverKnowWhyThis** [comment]
Another bout of minor tweaking, ears and tail seem fine for now.

Bigger issue is that I can't get any body markings to actually show up on the character, and have no idea how to fix it. It *should* be called correctly, if I look at other objects other users have added.

**Comment by NeverKnowWhyThis** [comment]
All sprites are selectable in character creation, colourable and generally ok. There are some minor clipping issues with existingclothing, but that's almost impossible to fix without fully destroying how the base looks.

Added 2 more ear styles based of suggestions as wel.

I'll make another pull for the fluff tail some other time, it's been fully deleted from the branch.

**Pull request merged**